selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/NavigationBar.java

```diff
@@ -81,12 +81,12 @@ public void clickOnMenu(MenuItem menuItem) {
     seleniumWebDriver.findElement(By.xpath(locator)).click();
   }
 
-  public String getMenuCounterValue(MenuItem menuItem) {
+  public int getMenuCounterValue(MenuItem menuItem) {
     String locator =
         String.format(Locators.MENU_ITEM_XPATH, menuItem.title) + Locators.COUNTER_XPATH;
     String counter = seleniumWebDriver.findElement(By.xpath(locator)).getText();
```

Comment:
Here we can use regex to extract number from text:
seleniumWebDriver.findElement(By.xpath(locator)).getText().replaceAll("\\s*[(]([0-9]+)[)]\\s*", "$1"))